libs/wingcompiler/src/compile.ts

```diff
@@ -9,6 +7,7 @@ import { copyDir, normalPath } from "./util";
 import { existsSync } from "fs";
 import { BuiltinPlatform } from "./constants";
 import { CompileError, PreflightError } from "./errors";
+import { Worker } from "worker_threads";
```

Comment:
Can you elaborate why is this needed?

(Every time I touched worker threads in Node I got bit in the ass at some point.). Node is not well designed for multi threading...

Reply:
Thanks for the context. God speed